Tests/StatsdClientTests/Stub|Mock|Fake|Spy/SpyScheme.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  SpyScheme.swift
+//  StatsdClient
+//
+//  Created by Nghia Tran on 10/15/17.
+//  Copyright © 2017 StatsdClient. All rights reserved.
+//
+
+import Foundation
+@testable import StatsdClient
+
+class SpyScheme: FlushScheme {
```

Comment:
why named it `Spy`?

Reply:
Spy Object in Testing means we'd like to check if some specific function is called or not. It has simple responsibility than Mock or Stub Object, which are more complex.

Reply:
We have many kinds of object in Testing, such as Spy, Mock, Tub, Fake, etc.

Reply:
Interesting.